python/cugraph/traversal/ms_bfs.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
Do we want a test for this?

Reply:
I see that it is not been implemented but exposed, is that right?
In that case, we could either raise error or not expose it to user

Reply:
Right, it is just the API proposal. I can raise an error.